testsuite/src/it/java/org/graalvm/tests/integration/utils/BuildAndRunCmds.java

```diff
@@ -135,7 +135,7 @@ public enum BuildAndRunCmds {
                     new String[]{"powershell", "-c", "\"Expand-Archive -Path test_data.txt.zip -DestinationPath target -Force\""}
                     :
                     new String[]{"unzip", "test_data.txt.zip", "-d", "target"},
-            new String[]{"native-image", "-H:GenerateDebugInfo=1", "-H:+PreserveFramePointer", "-H:-DeleteLocalSymbols",
+            new String[]{"native-image", "-H:GenerateDebugInfo=1", "-H:+PreserveFramePointer", "-H:-DeleteLocalSymbols", "-H:+TrackNodeSourcePosition",
```

Comment:
Could we add a comment explaining that `-H:+TrackNodeSourcePosition` is explicitly needed for GraalVM 23.0+ as https://github.com/oracle/graal/commit/67cc5c201bcbef1d4abe02c966fdd1c35fc583c2 removed it from the default options when debug info generation is being turned on?

Reply:
Done